vortex-api/src/main/java/mil/army/usace/hec/vortex/io/BilZipDataReader.java

```diff
@@ -76,20 +76,6 @@ public VortexData getDto(int idx) {
             }
         }
         return null;
-    }
+    } // Extended getDto(): Bil Zip
 
```

Comment:
Let's leave this as a private method in the child class for now. We can explore options using an interface.